### ISSUE TITLE
In public API vaccine parameter is not filtering the vaccine

### ISSUE THREAD

**Issue by rchawathearkin** [cowinapi/developer.cowin#41]
In official cowin app, the following API e.g.
https://cdn-api.co-vin.in/api/v2/appointment/sessions/calendarByDistrict?district_id=294&date=10-05-2021&vaccine=COVAXIN returns only centers with Covaxin, but in the public API tried in the browser
https://cdn-api.co-vin.in/api/v2/appointment/sessions/public/calendarByDistrict?district_id=294&date=10-05-2021&vaccine=COVAXIN the centers are not filtered by vaccine. Even Covishield are sent in response. Seems to be a bug.

**Comment by SehgalNeeraj** [comment]
These are protected APIs and NOT Public right now.
How are you authorized to use them?

**Comment by angsdey2** [comment]
No these are public APIs

**Comment by rchawathearkin** [comment]
As per https://apisetu.gov.in/public/marketplace/api/cowin/cowin-public-v2 these are public APIs with a rate limit restriction, and available for use via browser as per the comments in other threads here.

**Comment by SehgalNeeraj** [comment]
Let me rephrase. 
The API is public but takes only 2 parameters. The "vaccine" parameter is not supported in public API and I get Unauthorized access.
So, how are you able to use the "vaccine" parameter? 


> In official cowin app, the following API e.g.
> https://cdn-api.co-vin.in/api/v2/appointment/sessions/calendarByDistrict?district_id=294&date=10-05-2021&vaccine=COVAXIN returns only centers with Covaxin, but in the public API tried in the browser
> https://cdn-api.co-vin.in/api/v2/appointment/sessions/public/calendarByDistrict?district_id=294&date=10-05-2021&vaccine=COVAXIN the centers are not filtered by vaccine. Even Covishield are sent in response. Seems to be a bug.

**Comment by SehgalNeeraj** [comment]
> In official cowin app, the following API e.g.
> https://cdn-api.co-vin.in/api/v2/appointment/sessions/calendarByDistrict?district_id=294&date=10-05-2021&vaccine=COVAXIN returns only centers with Covaxin, but in the public API tried in the browser
> https://cdn-api.co-vin.in/api/v2/appointment/sessions/public/calendarByDistrict?district_id=294&date=10-05-2021&vaccine=COVAXIN the centers are not filtered by vaccine. Even Covishield are sent in response. Seems to be a bug.

The API is public, but it does not support the "vaccine" parameter. This is under development and same can be confirmed from the corresponding protected API signature.
Where do you see support for "vaccine" param in the Public API signature ?

**Comment by rchawathearkin** [comment]
Yeah looks like I read the documentation for protected API where vaccine parameter was available and since it was not working for public API, filed this issue. It just seems to ignore that parameter as of now and public API works from browser as expected. So closing this issue since the vaccine parameter is not supported in public API as of yet.
